### PR TITLE
Import from onnx

### PR DESCRIPTION
**Pull request by likholat**

### CODE REVIEW

**Comment by likholat** [comment]
@dkurt

**Comment by dkurt** [comment]
Remove mo_extensions folder

File: setup.py [hunk 1]
```diff
@@ -46,7 +46,7 @@
         "ismrmrd==1.9.1",
         "tensorboard>=2.5.0",
         "openvino-dev[onnx]==2021.4.2",
```
Comment (dkurt) [inline setup.py]: `"openvino==2021.4.2",`

**Review by dkurt — APPROVED** [review]
Let's merge it

**Pull request merged**